Sources/Services/AWSPolly/PollyClient.swift

```diff
@@ -481,3 +481,23 @@ extension PollyClient: PollyClientProtocol {
     }
 
 }
+
+extension PollyClient {
+    public func presignURLSynthesizeSpeech(input: SynthesizeSpeechInput, expiration: Foundation.TimeInterval) async throws -> ClientRuntime.URL {
+        let presignedURL = try await input.presignURL(config: config, expiration: expiration)
+        guard let presignedURL else {
```

Comment:
nit: you can combine this & the above statement

Reply:
changed as suggested.

Reply:
Reverted back to original, after discussion with David: https://github.com/awslabs/aws-sdk-swift/pull/1198#discussion_r1378110524

Reply:
Fine as-is if you prefer two steps.